### PR TITLE
Java FAQ update

### PR DESCRIPTION
**Pull request by nrgraham23**
Updates to the FAQ for recent changes to the Java bindings.

Signed-off-by: Nathaniel Graham <ngraham@lanl.gov>

@hppritcha  @jsquyres

### CODE REVIEW

File: faq/java.inc [hunk 1]
```diff
@@ -11,7 +11,10 @@ likely to find new uses beyond traditional HPC applications.";
 $q[] = "What MPI coverage is provided by the Java interface?";
 $anchor[] = "java_cover";
 
-$a[] = "Almost full MPI-3 coverage.";
+$a[] = "Complete MPI-3.1 coverage is provided with a few exceptions. 
+The bindings for the MPI_Neighbor_alltoallw and MPI_Ineighbor_alltoallw 
+functions are planned for release v2.1. Also excluded are functions that
```
Comment (jsquyres) [inline faq/java.inc]: I'd say "after v2.0.0" instead of promising a specific version (e.g., v2.1.0).

File: faq/java.inc [hunk 2]
```diff
@@ -11,7 +11,10 @@ likely to find new uses beyond traditional HPC applications.";
 $q[] = "What MPI coverage is provided by the Java interface?";
 $anchor[] = "java_cover";
 
-$a[] = "Almost full MPI-3 coverage.";
+$a[] = "Complete MPI-3.1 coverage is provided with a few exceptions. 
+The bindings for the MPI_Neighbor_alltoallw and MPI_Ineighbor_alltoallw 
+functions are planned for release v2.1. Also excluded are functions that 
+incorporate the concepts of explicit virtual memory addressing.";
```
Comment (jsquyres) [inline faq/java.inc]: Can you give an example or two of the functions that are excluded?

**Comment by hppritcha** [comment]
:+1:

**Comment by nrgraham23** [comment]
This PR still needs to be merged.  I do not have permissions to do it.

@hppritcha  @jsquyres

**Comment by jsquyres** [comment]
Thanks for the reminder!

**Comment by nrgraham23** [comment]
Thanks!

**Pull request merged**